float64 time
float64 x
float64[7] data1
float64[7] data2
float64[7] data3
